protos/channel_responses.proto

```diff
@@ -57,12 +58,22 @@ enum GeminiWalletState {
   GEMINI_ACCOUNT_KYC = 1;
 }
 
+message EthereumWallet {
+  string address = 1;
+}
+
+message SolanaWallet {
+  string address = 1;
+}
+
 message Wallet {
   oneof provider {
     UpholdWallet uphold_wallet = 1;
     PaypalWallet paypal_wallet = 2;
     BitflyerWallet bitflyer_wallet = 3;
     GeminiWallet gemini_wallet = 4;
+    EthereumWallet ethereum_wallet = 5;
```

Comment:
The client does not include these definitions - we should probably keep them in sync?